src/pages/contributing.md

```diff
@@ -1,43 +1,238 @@
+
+When describing a concept, you should first introduce what the concept is, where it is used in Minecraft, why it should be used, and how to use it. Each section within a concept should have a header. A section can also be broken into sub-sections if necessary. For example, each recipe within a cookbook can have a sub-section for ingredients and the recipe itself.
+
+If you need to refer to other concepts, the relevant page should be linked along with a summary and/or some example to understand the application.
+
+## Examples
+
+Code examples should generally be pseudocode-like objects meant to enhance the understanding of a modder. For this documentation, pseudocode-like refers to code blocks written in the structure and syntax of the desired language with comments used as placeholders for specific logic that the modder may choose to implement themselves. The code blocks do not necessarily need to be compilable, but each line should have valid syntax and structure of the desired language.
+
+When implementing a method, it is usually specific to the desired goal a modder is trying to achieve. As a guide, this documentation aims to be somewhat agnostic to a modder's specific goal, instead covering the general use case.
+
+Let's say we are using a method called `#applyDiscount` to take some value off the current price. Not everyone will implement the same logic within the method. So, the pseudocode can leave a comment mentioning what to do instead:
+
+```java
+// In some class
+
+public float applyDiscount(float price) {
+    float newPrice = price;
+    // Apply discount to newPrice
+    // ...
+    return newPrice;
+}
+```
+
+If the pseudocode is not explanatory enough to understand the concept, then a full code example can be used instead. A full code example should supply dummy values and explain what they represent.
+
+## Minor and Patch Changes
+
+If a change occurs between a minor or patch versions of NeoForge, then relevant changes in the documentation should be split into separate sections or put into tabs. This maintains the accuracy of the information depending on the version the modder is currently developing for.
+
+```md
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs defaultValue="latest">
+<!-- Value represents the last supported version of this method. -->
+<!-- Label represents the version range to display to the reader. -->
+<TabItem value="latest" label="Latest">
+
+<!-- Markdown here -->
+
+</TabItem>
+<!-- There must be an empty line before and after the Markdown text in a tab. -->
+<TabItem value="20.2.67" label="[20.2.35,20.2.67]"> 
+
+<!-- Markdown here -->
+
+</TabItem>
+<TabItem value="20.2.34" label="[20.2.0,20.2.34]">
+
+<!-- Markdown here -->
+
+</TabItem>
+</Tabs>
+```
```

Comment:
Could we have this be used as an example in this page? So the reader knows the expected output and format (and can see if what they write is correct or not).

Reply:
Sure, here you go